CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v0.8.1](https://github.com/puppetlabs/puppetlabs-splunk_hec/tree/v0.8.1) (2020-05-11)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-splunk_hec/compare/v0.8.0...v0.8.1)
+
+### UNCATEGORIZED PRS; GO LABEL THEM
```

Comment:
This line means the referenced PR doesn't have a label that would tell changelog generator what section it belongs in. `GO LABEL THEM` means you need to go and review that PR and add a label and re-run changelog generator prior to attempting merge this changelog.